app/assets/stylesheets/content/_select2.scss

```diff
@@ -126,3 +115,78 @@
     margin-left: 0;
   }
 }
+
+//@todo for select2 visibility in livingStyleGuide, need another solution  it will rewrite globally
+.livingstyleguide--example {
+  overflow: visible !important;
+}
+
+//$se2- select2 var prefix
+$se2-theme-selectable-color: #f8f8f8;
+//todo change to global theme color var
+//{bgThemeHighlight}
+$se2-theme-selectable-color-highlighted: #24b3e7;
+
+$se2-element-height: 33px;
+$se2-line-height: 30px;
+$se2-button-width: 28px;
+$se2-arrow-button-width: 18px;
+$se2-results-max-height: 500px;
+
+$se2-max-width: 350px;
```

Comment:
@vcostin not sure if we want to keep these rules around - IMO ui-select widths should always take up the same width as regular `<select>` elements / @floriank